java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java

```diff
@@ -193,6 +197,50 @@ public void testAllFields() throws Exception {
             + "  \"repeatedNestedEnum\": [\"BAR\", \"BAZ\"]\n"
             + "}",
         toJsonString(message));
+    assertEquals(
```

Comment:
Please add one dedicated test case (one test method) for the new option (instead of spreading it everywhere).